src/YAMLValidator.Tests/ValidatorTest.cs

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Json.Schema;
+using Microsoft.PowerPlatform.PowerApps.Persistence;
+
+namespace YamlValidatorTests;
+
+[TestClass]
```

Comment:
nit: if some tests are asserting similar things, you might consider grouping them into one and parametrizing them: https://learn.microsoft.com/en-us/visualstudio/test/how-to-create-a-data-driven-unit-test?view=vs-2022